tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -1048,6 +1051,12 @@ public:
       fColumn.fFirstElementIndex = firstElementIdx;
       return *this;
    }
+   RColumnDescriptorBuilder &SetSuppressedDeferred()
+   {
+      if (fColumn.fFirstElementIndex > 0)
+         fColumn.fFirstElementIndex = -fColumn.fFirstElementIndex;
```

Comment:
Maybe assert that `fColumn.fFirstElementIndex > 0`? (otherwise this function does nothing, which probably hints at another problem)